terraform/roks_with_odf/variables.tf

```diff
@@ -26,7 +26,7 @@ variable "resource_group" {
 }
 
 variable "roks_version" {
-  default     = "4.7"
+  type = string
```

Comment:
Go ahead and leave in a default for this.  4.7 is fine

Reply:
done